src/app/archive/archive_lib/processor.ml

```diff
@@ -506,7 +506,7 @@ module Block = struct
                       Coda_base.User_command.forget_check user_command_checked
                   }
                 in
-                ( user_command :: acc_user_commands
+                ( acc_user_commands @ [user_command]
```

Comment:
we need a sequence number across all transactions (user command, fee transfers, coinbase) in order to accurately recreate the ledger. Do we always apply the user commands before the fee transfers and coinbase? In that case we can use a different seq-number starting at zero for each. But if not we need to keep the ordering.